cookiecutter.json

```diff
@@ -6,6 +6,7 @@
     "package_dir_name": "{{ cookiecutter.project_name|replace(' ', '_')|replace('-', '_')|lower }}",
     "repo_name": "{{ cookiecutter.project_name|replace(' ', '-')|lower }}",
     "project_short_description": "Python package for doing science.",
+    "maximum_supported_python_version": ["3.10", "3.11", "3.12"],
     "minimum_supported_python_version": ["3.8", "3.9", "3.10"],
```

Comment:
see above.  We are only supporting 3.10 and above atm, so there doesn't seem much point to offer people 3.8/3.9?